service/create/service.go

```diff
@@ -94,17 +94,42 @@ func (s *Service) Boot() {
 	s.bootOnce.Do(func() {
 		s.Config.Logger.Log("info", "Booted cert-operator")
 
-		s.Config.Logger.Log("info", "Test issuing a cert")
-
 		cert := CertificateSpec{
-			ClusterID:        "cert-test",
-			CommonName:       "api.cert-test.giantswarm.io",
-			IPSANs:           []string{"10.0.0.4", "10.0.0.5"},
-			AltNames:         []string{"api.k8s.cert-test.giantswarm.io"},
-			AllowBareDomains: false,
+			ClusterID:  "g8s-cert-test-eu-west-1",
+			CommonName: "api.cluster-test.g8s.eu-west-1.aws.test.private.giantswarm.io",
+			IPSANs:     []string{"10.0.0.4", "10.0.0.5"},
+			AltNames: []string{
+				"kubernetes",
+				"kubernetes.default",
+				"kubernetes.default.svc",
+				"kubernetes.default.svc.cluster.local",
+			},
+			AllowBareDomains: true,
```

Comment:
These alt names are from kubernetesd. Since they include kubernetes allow bare domains will need to be true.